lib/screens/signup/signup.dart

```diff
@@ -1,7 +1,7 @@
 import 'package:ecellapp/screens/signup/cubit/signup_cubit.dart';
-import 'package:ecellapp/screens/signup/widgets/email_field.dart';
-import 'package:ecellapp/screens/signup/widgets/name_field.dart';
-import 'package:ecellapp/screens/signup/widgets/password_field.dart';
+import 'package:ecellapp/widgets/name_field.dart';
```

Comment:
`name_field.dart` is not used anywhere else, so there is no need to move it.

Reply:
shouldn't there be two separate fields for name, first name and second name?

Reply:
Pull the latest code from main. There are 2 files made by sid in that commit. You don't have to move that.